docs/development/pigs/pig-012.rst

```diff
@@ -0,0 +1,340 @@
+.. include:: ../../references.txt
+
+.. _pig-012:
+
+*****************************
+PIG 12 - High-level interface
+*****************************
+
+* Author: José Enrique Ruiz, Christoph Deil, Axel Donath, Regis Terrier, Lars Mohrmann
+* Created: June 6, 2019
+* Accepted: tbd
+* Status: draft
+* Discussion: `GH 2219`_
+
+Abstract
```

Comment:
Abstract has to say super-prominently and clearly (separate line):
```
We propose to add a single `Analysis` class that supports config-file driven analysis
of the main IACT source analysis use cases.
```
(maybe mention already "similar to Fermipy or HAP in HESS")